server/uploadHandler/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vtaa",
```

Comment:
Should the upload handler be name `vtaa`. Also its worth noting that the root `package.json` is named `visiondroid`